src/middleware/guard.ts

```diff
@@ -13,7 +13,7 @@ export const guard = (req: Request, res: Response, next: NextFunction) => {
         if (token) {
             jwt.verify(token, process.env.APPLICATION_SECRET, (err, user) => {
                 if (err) {
-                    console.log(err);
+                    this.logger.error(err.toString());
```

Comment:
this.logger isn't available here. You can import winston and use winston.logger instead of this.logger .

Reply:
oops had not seen this and thought logger is module from npm, just copy past